aboutus.htm

```diff
@@ -0,0 +1,3 @@
+<html>
+    <p>This is about us page.</p>
```

Comment:
pls add span tag dont use p tag thank you yours faithfully abhinav

Reply:
Go fuck yourself.